examples/dkr-create-oracle-xe-server

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# Variables used for the script
+if [ ! -f `which pwgen` ]; then
+  echo "You must have pwgen on your machine to run this script"
+  exit 1
+fi
+
+CMG_COMMAND=docker
+DKR_IMAGE="gvenzl/oracle-xe"
+DKR_HOST_PORT=$1
+DKR_CNT_NAME=$2
+DKR_DATA_PATH=dbcxe-$DKR_CNT_NAME-dvol
+DKR_ORA_USER_APP=DAAPPUSER
+DKR_ORA_PW=`pwgen -1 16 | tr -d '\n'`
+DKR_ORA_PW_APP=`pwgen -1 16 | tr -d '\n'`
+
+echo "Docker Image ...............:'$DKR_IMAGE'"
+echo "Docker User ................:'$DKR_ORA_USER_APP'"
```

Comment:
Should that one not say "Database App User". Docker User could be confusing to some.